vct/kustomize/vct/overlays/local/vct3/sedb64transform.yml

```diff
@@ -8,4 +8,4 @@ apiVersion: svceng
 kind: SedB64Transformer
 metadata:
   name: sedb64transformer
-argsOneLiner: s^||COUCHDB_URL||^${COUCHDB_URL}^ s^||TRILLIAN3_LOG_MYSQL||^${TRILLIAN3_LOG_MYSQL}^
+argsOneLiner: s^||COUCHDB_URL||^${COUCHDB_URL}^ s^||TRILLIAN3_LOG_MYSQL||^${TRILLIAN3_LOG_MYSQL}^ s^||TRILLIAN3_LOG_POSTGRES||^${TRILLIAN3_LOG_POSTGRES}^
```

Comment:
need to remove TRILLIAN3_LOG_MYSQL

Reply:
@fqutishat I did not remove it on purpose. What if we want to switch back for some reason?